### PR TITLE
Fix migration name on modified date migration

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: readthedocs/projects/migrations/0031_add_modified_date_importedfile.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('projects', '0029_add_additional_languages'),
+        ('projects', '0030_add_modified_date_importedfile'),
```
Comment (stsewd) [inline readthedocs/projects/migrations/0031_add_modified_date_importedfile.py]: Ups, I approved, but this should be `0030_change-max-length-project`

Comment (ericholscher) [reply readthedocs/projects/migrations/0031_add_modified_date_importedfile.py]: ha, indeed _sigh_

Comment (ericholscher) [reply readthedocs/projects/migrations/0031_add_modified_date_importedfile.py]: Good thing CI caught it :D

File: readthedocs/projects/migrations/0031_add_modified_date_importedfile.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('projects', '0029_add_additional_languages'),
+        ('projects', '0030_change-max-length-project'),
```
[the remark this answers is not in the document]
Comment (stsewd) [reply readthedocs/projects/migrations/0031_add_modified_date_importedfile.py]: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/projects/migrations/0030_change-max-length-project-slug.py

[the remark this answers is not in the document]
Comment (ericholscher) [reply readthedocs/projects/migrations/0031_add_modified_date_importedfile.py]: _quietly rewrites history like it never happened_

**Pull request merged**